.github/workflows/deploy-template.yml

```
@@ -70,6 +70,16 @@ jobs:
           release="${{ steps.parse_directories.outputs.release }}"
           CURL_CMD="curl -u $url_part:${{ secrets.CLOUD_PASSWORD }}"
 
+          case "$model" in
+            *vp2430*|*vp2440*|*vp3230*|*3210*)
+              extra_precision="$model"
```

```suggestion
              subdir="$model"
```

and all other instances

---

ok here: https://github.com/Dasharo/coreboot/pull/675/commits/0ffc78197b79a11df5dba103cd1a7fb69a3188bc